packages/design-system/src/utilities.ts

```diff
@@ -94,9 +94,9 @@ export const useResize = ({
   onResizeEnd,
   timeout = 300,
 }: {
-  onResizeStart?: () => void;
-  onResize?: () => void;
-  onResizeEnd?: () => void;
+  onResizeStart?: (entries: Array<ResizeObserverEntry>) => void;
+  onResize?: (entries: Array<ResizeObserverEntry>) => void;
+  onResizeEnd?: (entries: Array<ResizeObserverEntry>) => void;
```

Comment:
where entries are used?

Reply:
I was in the middle of refactoring and used them, then removed the usage, thought its still useful potentially since the utility is generic.